SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -70,3 +70,5 @@ extension Station: VOKMappableModel {
         }
     }
 }
+
+extension Station: VokoderTypedManagedObject { }
```

Comment:
(I see the note in the actual protocol, but it might not hurt to put something in with the code as well to be 100% clear)